packages/transmute-framework/src/stream-model/index.js

```diff
@@ -62,7 +62,7 @@ module.exports = class StreamModel {
     if (eventCount === 0) {
       return;
     }
-    if (this.state.lastIndex == null || this.state.lastIndex < eventCount) {
+    if (this.state.lastIndex == null || this.state.lastIndex < eventCount - 1) {
```

Comment:
either check if lastIndex is `null` or `undefined` or `""` with `if(!this.state.lastIndex || ...)` or check if lastIndex is `null` specifically with `if(this.state.lastIndex === null || ...)`